cmd/ocm-backplane/login/login.go

```diff
@@ -261,10 +262,10 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 
 	logger.Debugln("Extracting backplane URL")
 	// Get Backplane URL
-	bpURL := globalOpts.BackplaneURL
-	if bpURL == "" {
-		bpURL = bpConfig.URL
+	if globalOpts.BackplaneURL != "" {
+		bpConfig.URL = globalOpts.BackplaneURL
```

Comment:
The problem is that the bpConfig.URL is used later on and I assume that users expect if you provide a new backplane URL on the CLi it overrides the one in the config everywhere. The other option of course is a much bigger refactor so that the bpConfig object is created with the correct URL when an override is given on the command line.

Reply:
Reverted this change